create.go

```diff
@@ -127,12 +128,60 @@ func (opts *CreateOptions) Resolve(ctx context.Context) (*exec.Cmd, error) {
 		opts.closers = append(opts.closers, cancel)
 	}
 
+	outSenders := []send.Sender{}
+	errSenders := []send.Sender{}
+	for _, logger := range opts.Output.Loggers {
+		sender, err := logger.LogType.Configure(logger.LogOptions)
+		if err != nil {
+			return nil, err
+		}
+		if !logger.LogOptions.IgnoreOutput {
+			outSenders = append(outSenders, *sender)
+		}
+		if !logger.LogOptions.IgnoreError {
+			errSenders = append(errSenders, *sender)
+		}
+	}
+
+	if len(outSenders) == 0 {
+		devNull, err := makeDevNullSender()
+		if err != nil {
+			return nil, err
+		}
+		outSenders = append(outSenders, devNull)
+	}
+	if len(errSenders) == 0 {
+		devNull, err := makeDevNullSender()
+		if err != nil {
+			return nil, err
+		}
+		errSenders = append(errSenders, devNull)
+	}
+
+	outMulti, err := send.NewMultiSender(DefaultLogName, send.LevelInfo{Default: level.Info, Threshold: level.Trace}, outSenders)
+	if err != nil {
+		return nil, err
+	}
+	errMulti, err := send.NewMultiSender(DefaultLogName, send.LevelInfo{Default: level.Error, Threshold: level.Trace}, errSenders)
+	if err != nil {
+		return nil, err
+	}
+
+	opts.Output.OutputSender = send.NewWriterSender(outMulti)
+	opts.Output.ErrorSender = send.NewWriterSender(errMulti)
+
 	cmd := exec.CommandContext(ctx, opts.Args[0], args...) // nolint
 	cmd.Dir = opts.WorkingDirectory
-	cmd.Stderr = opts.Output.GetError()
-	cmd.Stdout = opts.Output.GetOutput()
+	cmd.Stdout = io.MultiWriter(opts.Output.GetOutput(), opts.Output.OutputSender)
+	cmd.Stderr = io.MultiWriter(opts.Output.GetError(), opts.Output.ErrorSender)
```

Comment:
shouldn't opts.Output.GetOutput() and opts.Output.OutputSender (ditto for error) be the same thing? or at least, it might be possible to elided Output.OutputSender by moving lines 131-172 into GetOutput/GetError (refactored as needed.)

Reply:
I changed these two lines to just call `GetOutput()` and `GetError()` but is it okay that `OutputOptions.Output != OutputOptions.GetOutput()` (and in the same vein, `OutputOptions.Error != OutputOptions.GetError()`)?

Reply:
I think I'm missing a patch

Reply:
Just pushed it.